### ISSUE TITLE
Xbox one controller

### ISSUE THREAD

**Issue by colhemm** [nefarius/ViGEm#1]
Hi nefarius,
A very helpful guy could shauleiz sent me over to you and said you might be able to help me.

I can hold a joystick but I can't hold a joypad and I need to control the buttons with my voice. The Oculus Rift will only let you use an Xbox one controller so that's me out.

A great guy called `` billyisms helped me setup something with XOutput, vJoy and UCR. It really helped but it only creates an Xbox 360 controller.

**Comment by nefarius** [comment]
Xbox One Controller emulation is in the works :smile:

**Comment by colhemm** [comment]
Great news ,Thank you so much for your help I can't wait to try it out.

**Comment by nefarius** [comment]
I am so close, I can feel it. It needs to stop resisting me :angry:

**Comment by nefarius** [comment]
[Har har har!](https://twitter.com/CNefarius/status/761677011781910528) We have a winner! :smile:

**Comment by colhemm** [comment]
Wow! great news have you done it :

**Comment by colhemm** [comment]
If it's ok I'll need you to tell me how to install it, and thank you sooooo much for making my life a lot easier and happier.

**Comment by nefarius** [comment]
This is currently halted since a recent Windows Update broke things on my end but I will continue soon.

**Comment by aa8y** [comment]
@nefarius Did this ever happen?

**Comment by nefarius** [comment]
Nope, reversing this b*tch took more time and effort than I expected so I prioritized on other things.
